### PR TITLE
chore: improve README's readability a bit

### PR DESCRIPTION
**Pull request by victorteodoro**
Some changes to correct typos and make the text a bit more idiomatic.

Love,
Thanks

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,12 +30,11 @@ $ npm install --save-dev eslint@4.3.0 \
 ```
 
 > The peer dependencies specified above have hardcoded versions.
-> If you prefer you can use the command
+> If you prefer you, can use the command
```
Comment (mccraveiro) [inline README.md]: Isn't this comma in the wrong place?

File: README.md [hunk 2]
```diff
@@ -104,7 +102,7 @@ AwesomeCard/
 
 # Project organization
 
-Components should be divided in at least three directories:
+Your project components should be seprated in at least three directories:
```
Comment (mccraveiro) [inline README.md]: Typo: separated

File: README.md (hunk 1 above)
Comment (victorteodoro) [reply README.md]: Yep

**Comment by derekstavis** [comment]
Thanks a lot and sorry for missing this ou!t!! :beers:

**Pull request merged**